homeassistant/components/device_automation/__init__.py

```diff
@@ -36,19 +36,31 @@
     }
 )
 
+
+class DeviceAutomationDetails(NamedTuple):
+    """Details for device automation."""
+
+    platform_name: str
```

Comment:
Using platform here is confusing, can we change this to `section` or `automation_section`?